documentation/docs/pages/operators/nodes/performance-and-testing/node-api-check.mdx

```diff
@@ -0,0 +1,160 @@
+
+Besides querying any bonded node APIs, `nym_api_check.py` has a function counting all existing nodes in provided version.
+
+### Setup
+
+#### Pre-requsities
+
+<Steps>
+
+###### 1. Install and configure Python3
+
+
+- Start with installing Python3:
+```sh
+sudo apt install python3
+```
+
+- Make sure Python3 is your default Python version:
+```sh
+update-alternatives --install  /usr/bin/python python /usr/bin/python3 1
+```
+
+- Verify:
+```sh
+python --version
+
+# should return higher than 3
+```
+
+- Install Python modules `tabulate`, `pandas` and `argparse` either using [`pip`](https://python.land/virtual-environments/installing-packages-with-pip) or if you installed Python3 system-wide you can install modules directly:
+
+<div>
+  <Tabs items={[
+    <strong>Using <code>pip</code></strong>,
+    <strong>System-wide Python installation</strong>,
+    ]} defaultIndex="1">
+    <MyTab>
+      ```sh
+      pip install tabulate pandas argparse
+      ```
+    </MyTab>
+    <MyTab>
+      ```sh
+      sudo apt install python3-tabulate python3-pandas python3-argparse -y
+      ```
+    </MyTab>
+  </Tabs>
+</div>
+
+###### 2. Install `node_api_check.py` and make executable  
+
+To run the program you neet to have  [`node_api_check.py`](https://github.com/nymtech/nym/tree/develop/scripts/node_api_check.py) and [`api_endpoints.json`](https://github.com/nymtech/nym/tree/develop/scripts/api_endpoints.json). 
+
+- If you [compiled from source](../../binaries/building-nym.mdx), you already have both of these files. Note that the latest version of this program may be on `develop` branch.
+
+- If you prefer to download them individually, do it by opening terminal in your desired location and running:
+```sh
+wget https://raw.githubusercontent.com/nymtech/nym/tree/develop/node_api_check.py
+
+wget https://raw.githubusercontent.com/nymtech/nym/tree/develop/api_endpoints.json
+```
+
+- Make executable:
+```sh
+chmod u+x node_api_check.py
+```
+
+</Steps>
+
+Now you are ready to check your node.
+
+### Usage & Commands
+
+- Run the program with `--help` flag to see the available commands:
+```sh
+./node_api_check.py --help
+```
+
+- Output:
+```python
+<!--cmdrun cd ../../../../scripts && python ./node_api_check.py --help-->
+```
+
+#### `query_stats`
+
+When you want to see all the options connected to any command, add a `--help` flag after the command of your choice, like in this example:
+```sh
+./node_api_check query_stats --help
+```
+
+```python
+<!--cmdrun cd ../../../../scripts && python ./node_api_check.py query_stats --help-->
```

Comment:
also remove?

Reply:
I left all cmdrun for easier location post merge